frontend/src/components/task.jsx

```diff
@@ -22,16 +22,13 @@ export default function Task(props) {
     };
 
     const deleteTask = async (taskId) => {
-        const res = await fetch(`http://localhost:3001/tasks/${taskId}`, {
+        await fetch(`http://localhost:3001/tasks/${taskId}`, {
```

Comment:
After my changes, we are not reading the response anymore, so I removed the `res` variable here and just called the endpoint.